automation/validate-pvc-name-stability.py

```diff
@@ -0,0 +1,105 @@
+#! python
+
+import logging
+import os
+import os.path
+import yaml
+import subprocess
+import sys
+
+from kubernetes import client, config
+from openshift.dynamic import DynamicClient
+
+
+def validatePVCNames(path, liveTemplates):
+    templates = [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
+    errors = []
+    for templateFilename in templates:
+        with open(os.path.join(path, templateFilename), 'r') as stream:
+            try:
+                template = yaml.safe_load(stream)
+
+                if template == None:
+                    logging.info("Empty template file: %s", template)
+                    continue
+
+                templateName = template["metadata"]["name"]
+                logging.info("Checking PVC name stability for: {}".format(templateName))
+
+                pvcName = getPVCNameFrom(template)
+                pvcNamespace = getPVCNamespaceFrom(template)
+
+                matchingLiveTemplate = liveTemplates.get(templateName, None)
```

Comment:
The second argument to `get()` is `None` by default, so you don't need to write it explicitly.